fastestimator/pipeline.py

```diff
@@ -62,16 +62,17 @@ class Pipeline:
     ops: List[Union[NumpyOp, Scheduler[NumpyOp]]]
```

Comment:
let's add a hint here for data as well since it's getting so nested

Reply:
added hint for data